t/node/rr-balance.t

```diff
@@ -246,3 +246,80 @@ GET /t
 [{"count":2,"port":"1982"},{"count":4,"port":"1981"},{"count":6,"port":"1980"}]
 --- no_error_log
 [error]
+
+
+
+=== TEST 7: set route(weight is 0)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "uri": "/server_port",
+                    "upstream": {
+                        "type": "roundrobin",
+                        "nodes": {
+                            "127.0.0.1:1980": 3,
+                            "127.0.0.1:1981": 0,
```

Comment:
the admin API will return `127.0.0.1:1981` or not? we need add test cases for that.

Reply:
https://github.com/iresty/apisix/pull/536/files#diff-3d55f969c5c7e4abd188c26251844464R750

added new test case.